src/main/java/gyro/aws/codebuild/ReportGroupResource.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2020, Brightspot.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package gyro.aws.codebuild;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import gyro.aws.AwsResource;
+import gyro.aws.Copyable;
+import gyro.core.GyroUI;
+import gyro.core.Type;
+import gyro.core.resource.Id;
+import gyro.core.resource.Output;
+import gyro.core.resource.Resource;
+import gyro.core.resource.Updatable;
+import gyro.core.scope.State;
+import gyro.core.validation.Required;
+import gyro.core.validation.ValidStrings;
+import software.amazon.awssdk.services.codebuild.CodeBuildClient;
+import software.amazon.awssdk.services.codebuild.model.BatchGetReportGroupsResponse;
+import software.amazon.awssdk.services.codebuild.model.CreateReportGroupResponse;
+import software.amazon.awssdk.services.codebuild.model.InvalidInputException;
+import software.amazon.awssdk.services.codebuild.model.ReportGroup;
+import software.amazon.awssdk.services.codebuild.model.ResourceNotFoundException;
+import software.amazon.awssdk.services.codebuild.model.Tag;
+
+@Type("report-group")
+public class ReportGroupResource extends AwsResource implements Copyable<BatchGetReportGroupsResponse> {
```

Comment:
BatchGetReportGroupsResponse is not the correct model, it has a list of the resource that you want.

This change will require the resource as well as the finder refactor

```suggestion
public class ReportGroupResource extends AwsResource implements Copyable<ReportGroups> {
```